js/script.js

```diff
@@ -892,21 +940,23 @@ $(function () {
                 addNumberButton(numberButton);
             }
             rp.session.loadingNextImages = false;
-            
+
         };
 
         if (rp.settings.debug)
             console.log('Ajax requesting: ' + jsonUrl);
 
-        // I still haven't been able to catch jsonp 404 events so the timeout
-        // is the current solution sadly.
-        // Note we're still using `jsonp` despite potential issues because
-        // `http://www.redditp.com/r/randnsfw` wasn't working with CORS for some reason.
-        // https://github.com/ubershmekel/redditp/issues/104
```

Comment:
You removed this comment, without providing an explanation for the `jsonp` toggling or why did you make the following specific exceptions to`useJsonP`. We should pick this up in a future PR.

Reply:
Sure, it seemed to me that useJsonP with the code, that decides whether it's true or false, is pretty self-explanatory, but as I said, I expect this to be solved better when we tackle the redirected subreddits.